src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/interceptors/SessionCookieInterceptor.kt

```diff
@@ -22,7 +22,7 @@ class SessionCookieInterceptor(private val properties: SessionCookieProperties)
     request.cookies?.firstOrNull { it.name == SPRING_SESSION_COOKIE }?.apply {
       this.maxAge = properties.maxAge.toSeconds().toInt()
       this.path = ANY_PATH_STARTING_AT_ROOT
-      this.secure = request.isSecure
```

Comment:
Documentation says this will be determined by the request type e.g. secure HTTPS.  This did not appear to be the case after looking at the deployment on DEV>

Reply:
Do we think this is perhaps because the load balancer in front of the app server terminates the HTTPS? I wonder if you could configure the app to pick up the value from the (non-standard, but pretty common) [`X-Forwarded-Proto` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto)?

Reply:
Too be honest (hence the PR) I am not sure.  I didn't investigate that thoroughly.  I am sure the docs are probably right and it is more environmental like you say.